designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -132,3 +137,25 @@ trait DeploymentSynchronisationSupported extends DeploymentSynchronisationSuppor
 }
 
 case object NoDeploymentSynchronisationSupport extends DeploymentSynchronisationSupport
+
+sealed trait PeriodicExecutionSupport
+
+trait PeriodicExecutionSupported extends PeriodicExecutionSupport {
+
+  def engineHandler(
+      modelData: BaseModelData,
+      dependencies: DeploymentManagerDependencies,
+      deploymentConfig: Config,
+  ): PeriodicDeploymentEngineHandler
```

Comment:
Let's not introduce new phrases such as `deployment engine`.

Reply:
Name changed to `ScheduledExecutionPerformer`. It is a small interface, that the DeploymentManager must support, in order to perform scheduled executions (actually run scenario on a specific engine). Described in a comment above.